src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs

```diff
@@ -31,18 +31,18 @@ namespace System.Net.Quic;
 /// </summary>
 /// <remarks>
 /// <see cref="QuicConnection" /> can either be accepted from <see cref="QuicListener.AcceptConnectionAsync(CancellationToken)" /> (inbound connection),
-/// or create with a static method <see cref="QuicConnection.ConnectAsync(System.Net.Quic.QuicClientConnectionOptions, CancellationToken)" /> (outbound connection).
+/// or create with a static method <see cref="ConnectAsync(QuicClientConnectionOptions, CancellationToken)" /> (outbound connection).
```

Comment:
why? I think this is needed so generated documentation is linked properly. 
@ManickaP  may know for sure....

Reply:
The compiler suggests these changes.